packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -191,9 +191,11 @@ export interface IBasePickerProps<T> extends React.Props<any> {
   onInputChange?: (input: string) => string;
 
   /**
-   * A callback to override the default behavior of adding the selected suggestion on dismiss.
+   * A callback to override the default behavior of adding the selected suggestion on dismiss. If it returns true or
```

Comment:
Given the earlier comments I might change this to `if it returns true or undefined` rather than `nothing` to be more explicit. But this is not required.